lib/qesapdeployment.pm

```diff
@@ -353,7 +354,8 @@ sub qesap_execute {
 
     my $verbose = $args{verbose} ? "--verbose" : "";
     my %paths = qesap_get_file_paths();
-    my $exec_log = "/tmp/qesap_exec_" . $args{cmd} . ".log.txt";
+    my $exec_log = "/tmp/qesap_exec_$args{cmd}_$args{cmd_options}.log.txt";
+    $exec_log =~ s/[-\s]+/_/g;
```

Comment:
Thanks for the log name fix !
And you even included the cmd option :+1:

Reply:
It is about fixing the problem of
```
qesap_execute(cmd => 'terraform', verbose => 1, timeout => 1200);
```
and
```
qesap_execute(cmd => 'terraform', cmd_options => '-d', verbose => 1, timeout => 1200);
```
Producing a log file with the same name. 
My proposed solution works but it is not optimal for commands that are not using **cmd_options**, like http://openqaworker15.qa.suse.cz/tests/49519/logfile?filename=cluster_configure-qesap_exec_configure_.log.txt (do you see, the underscore at the end)